package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "dev": "NODE_ENV=development webpack --progress --watch",
-    "build": "NODE_ENV=production webpack --progress && md5sum dist/*.js dist/*.html > dist/checksums.md5"
+    "build": "NODE_ENV=production webpack --progress && cd dist && find -type f -exec md5sum {} \\;|sort -s > checksums.md5"
```

Comment:
`find...` filter for `.html` and `.js`?

Reply:
Changed so the output path in md5 doesn't have the "dist" so you can compare checksum easier. Everything is relative to the current directory.  Also opened up to any files checksums  just in case...

Reply:
Changed so the output path in md5 doesn't have the "dist" so you can compare checksum easier. Everything is relative to the current directory. Also opened up to any files checksums just in case...